track.py

```diff
@@ -91,14 +114,29 @@ def main() -> None:
     except IOError as ioe:
         log.error(ioe)
```

Comment:
My bad, I forgot to add a return statement, otherwise we continue on with the programm.

```suggestion
        log.error(ioe)
        return
```